### ISSUE TITLE
PEP 621

### ISSUE THREAD

**Issue by dbanty** [David-OConnor/pyflow#145]
I learned about this project recently while reading up on [PEP 621](https://www.python.org/dev/peps/pep-0621/#dependencies-optional-dependencies) and [PEP 665](https://www.python.org/dev/peps/pep-0665/)(draft).

Since there's an official way to represent most of the info in the `[tool.pyflow]` sections, will this project support that method? Currently I'm using Poetry and it seems that there is a ton of push-back on the new format for some reason.

I'd be happy to contribute the support if it's welcome but just not planned.

**Comment by David-OConnor** [comment]
I agree with that PEP, and think it would make a welcome change. Unfortunately, I don't have enough time to implement. Willing to review and merge PRs though.

**Comment by messense** [comment]
FYI, I wrote [pyproject-toml-rs](https://github.com/PyO3/pyproject-toml-rs) for maturin which may be useful for pyflow as well.
